servant-minimalist.hsfiles

```diff
@@ -0,0 +1,232 @@
+{-# START_FILE package.yaml #-}
+name:                {{name}}
+version:             0.1.0.0
+github:              "{{github-username}}{{^github-username}}githubuser{{/github-username}}/{{name}}"
+license:             BSD3
+author:              "{{author-name}}{{^author-name}}Author name here{{/author-name}}"
+maintainer:          "{{author-email}}{{^author-email}}example@example.com{{/author-email}}"
+copyright:           "{{copyright}}{{^copyright}}{{year}}{{^year}}2019{{/year}} {{author-name}}{{^author-name}}Author name here{{/author-name}}{{/copyright}}"
+
+extra-source-files:
+  - README.md
+  - ChangeLog.md
+
+# Metadata used when publishing your package
+# synopsis:            Short description of your package
+# category:            {{category}}{{^category}}Web{{/category}}
+
+# To avoid duplicated efforts in documentation and dealing with the
+# complications of embedding Haddock markup inside cabal files, it is
+# common to point users to the README.md file.
+description:         Please see the README on GitHub at <https://github.com/{{github-username}}{{^github-username}}githubuser{{/github-username}}/{{name}}#readme>
+
+dependencies:
+  - base >= 4.7 && < 5
+  - aeson
+  - servant-server
```

Comment:
There should be proper bounds, on all dependencies, especially on `servant`. We do break something every major release.

Reply:
This is the moment my newbiness shines 👼 : do you have any idea how I can generate (or find) which dependency boundaries to enforce?

Reply:
Also: since Stack uses the Resolver/snapshot mecanism, do we really need to enforce boundaries in the Hpack file? Maybe this is only needed for Cabal file?

Reply:
We need to enforce the boundaries, because there are also others then `stack` users.

This is quite important, because when we get a bug report with `stack` &`hpack` only based reproducion case (and I'd expect this template to be used to generate ones), I'd still need to do manual work, because I'm not using `stack`.

For that reason, I'd like the template to always explicitly use the latest released version of `servant`. If there's even a small change, that the user will find out that an issue isn't reproducible with the latest version, it's a win.